bubblesub/ui/views.py

```diff
@@ -0,0 +1,168 @@
+# bubblesub - ASS subtitle editor
+# Copyright (C) 2018 Marcin Kurczewski
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import enum
+
+from PyQt5 import QtCore, QtWidgets
+
+from bubblesub.api import Api
+
+
+class ViewLayout(enum.Enum):
+
+    Full = "full"
+    Audio = "audio"
+    Video = "video"
+    Subs = "subs"
```

Comment:
I thought we had agreed to stop using this and I see it for like third time

Reply:
We had agreed to move it out the GUI API, not make it disappear. If I remove that, I think it would be messy the restore/store of a view.

Reply:
Let me be clear, I could remove that, but it would be messy imho